src/config/app.config.ts

```diff
@@ -10,8 +10,13 @@ export const appConfig = () => ({
     password: process.env.DB_PASSWORD,
     database: process.env.DB_NAME,
     entities: getMetadataArgsStorage().tables.map((tbl) => tbl.target),
-    synchronize: true, // IMPORTANT: Turn this off on Production
+    synchronize: false, // IMPORTANT: Turn this off on Production
```

Comment:
Please don't disable this just yet, we need the db to be synchronized since we are still on development. Once we go live we should be very careful on not forget to disable it though.

Reply:
I will make a enviroment variable to handle this